graphql/testdata/patch1.graphql

```diff
@@ -0,0 +1,10 @@
+{
+  userPatches(userId: "testuser") {
+    id
+    variants
+    variantsTasks {
+      name
+      tasks
+    }
+  }
```

Comment:
why only testing these fields as opposed to all fields in type `Patch` ?

Reply:
this represents a query that could be made from the frontend and we will probably copy/paste query strings from Spruce to test

Reply:
yes that part is clear. my question is why not include all the Patch fields as opposed to just some fo them

Reply:
I picked some arbitrary fields. I changed it to include all fields